gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
-version=1.3.2-SNAPSHOT
-ballerinaLangVersion=2201.2.0
+version=1.4.0-SNAPSHOT
+ballerinaLangVersion=2201.4.0-20221116-082400-3eca35ee
```

Comment:
Shall we have `2201.4.0-20221208-193400-e9fb14ee`? This is the latest.

Reply:
done